LibRangeCheck-3.0/LibRangeCheck-3.0.lua

```diff
@@ -628,6 +610,46 @@ local function findSpellIdx(spellName)
   return nil
 end
 
+local function fixRange(range)
+  if range then
+    return math_floor(range + 0.5)
+  end
+end
+
+local function getSpellData(sid)
+  local name, _, _, _, minRange, range = GetSpellInfo(sid)
+  return name, fixRange(minRange), fixRange(range), findSpellIdx(name)
+end
+
+local function findMinRangeChecker(origMinRange, origRange, spellList)
+  for i = 1, #spellList do
+    local sid = spellList[i]
+    local name, minRange, range, spellIdx = getSpellData(sid)
+    if range and spellIdx and origMinRange <= range and range <= origRange and minRange == 0 then
+      return checkers_Spell[findSpellIdx]
+    end
+  end
+end
+
+local function getCheckerForSpellWithMinRange(spellIdx, minRange, range, spellList)
+  local checker = checkers_SpellWithMin[spellIdx]
+  if checker then
+    return checker
+  end
+  local minRangeChecker = findMinRangeChecker(minRange, range, spellList)
+  if minRangeChecker then
+    checker = function()
```

Comment:
I think there's a missing "unit" parameter here.

Reply:
Indeed, thanks.  Fixed in 71b6998